modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php

```diff
@@ -49,6 +49,53 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
         }
       }
     }
+
+    // Text Background Color.
+    if (!empty($this->configuration['bg_text_color'])) {
+      $bg_text_color = $row->getSourceProperty($this->configuration['bg_text_color']);
+      $bg_text_color_mapping = [
+        'bg-transparent' => 'transparent',
+        'bg-trans-white' => 'light',
+        'bg-trans-sky' => 'light',
+        'bg-trans-arizona-blue' => 'dark',
+        'bg-trans-black' => 'dark',
+      ];
+      foreach ($bg_text_color as $bg_text_color_item) {
+        $behavior['bg_color'] = $bg_text_color_mapping[$bg_text_color_item['value']];
+      }
+    }
+
+    // Text Background Color.
```

Comment:
Update comments

Reply:
done

Reply:
Looks like these comments still haven't been updated.  I think we might have been getting mixed up between this one and the full-width background one...

Reply:
I'm not seeing where this is done.. This line still says `// Text Background Color.` which is incorrect.